torch_ttnn/utils.py

```diff
@@ -9,14 +9,19 @@ def GraphCleanup(gm: torch.fx.GraphModule) -> torch.fx.GraphModule:
     return gm
 
 
-def get_shape(node_or_shape):
+def get_shape(gm: torch.fx.GraphModule, node_or_shape):
```

Comment:
I think this function deserves for some doc string

Reply:
agree. I have updated it for doc string b280d540a5e5c31bde8e75d0f312c1b2815419f8